### PR TITLE
Rockets

### PR DESCRIPTION
**Pull request by shubhambhoyar077**
## In this PR following project requirement is implemented:-

- [x] Created basic structure for Rockets - Setup #41 
- [x]  Fetched rockets - Fetch data #37
- [x] Displaying rockets - Lists render #33
- [x] Implemented rocket booking - Actions #32
- [x] Implemented rocket booking cancelation - Actions #28
- [x] Switch badges for Rockets - Conditional components #25

- [x] Added styling for NavBar.
- [x] Fix linters

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
         </div>
         <ul>
-          <NavLink>Rockets</NavLink>
-          <NavLink>Dragons</NavLink>
-          <NavLink>Mission</NavLink>
-          <NavLink to="my_profile" >My Profile</NavLink>
+          <li><NavLink to="/">Rockets</NavLink></li>
+          <li><NavLink>Dragons</NavLink></li>
+          <li><NavLink>Mission</NavLink></li>
+          <li><NavLink to="my_profile">My Profile</NavLink></li>
```
Comment (codedit334) [inline src/components/NavBar.js:18]: Consider adding `activeClassName="active"` to NavLink

File: src/components/rockets/Rockets.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import RocketLi from './RocketLi';
+import '../../styles/rockets/rocket.css';
+
+export default function Rockets() {
+  const { rockets, isRocketLoading } = useSelector((state) => state.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!rockets.length) {
+      dispatch(getRockets());
+    }
```
Comment (codedit334) [inline src/components/rockets/Rockets.js:13]: As mentioned in your task, data should be fetched at the start of the webapp:
fetchWhenStarts
- Consider adding `dispatch(getRockets)` at `/index.js`

**Review by codedit334 — CHANGES_REQUESTED** [review]
## Hi @shubhambhoyar077, 
it's nice that you've progressed this far, but there is some changes to do so we get approved on the first try 😉

File: src/components/rockets/Rockets.js (hunk 2 above)
Comment (shubhambhoyar077) [reply src/components/rockets/Rockets.js:13]: Our `dispatch(getRockets)` are link as [index](https://reactrouter.com/en/main/route/route#index) in routes so its get called first automatically whenever we render index.js or App.js.
Screenshot from 2023-04-10 21-50-11

File: src/components/NavBar.js (hunk 1 above)
Comment (codedit334) [reply src/components/NavBar.js:18]: Thank you for your clarification @shubhambhoyar077, I had an outdated response from the internet... Then please my only the active one underlined 

> Use <NavLink /> for the page navigation links and style active class to indicate which section/page user is currently on (underline active navigation link).

**Pull request merged**